pkgs/tools/networking/debian-hostname/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "debian-hostname";
+  version = "3.23";
+
+  src = fetchurl {
+    url = "http://deb.debian.org/debian/pool/main/h/hostname/hostname_${version}.tar.gz";
+    sha256 = "sha256-vG0ZVLIoSYaf+LKmAuOfCLFwL2htS1jdeSfN61tIdu8=";
+  };
+
+  outputs = ["out"];
+
+  patches = [];
+
+  nativeBuildInputs = [];
+  buildInputs = [];
+  doCheck = false;
+  installFlags = [];
+  installPhase = ''
+    sed -i.orig '/install/ s,-[og] root,,g' Makefile
```

Comment:
No need to backup the original Makefile. Maybe, the following would be more readable:
```sh
substituteInPlace Makefile --replace 'install -o root -g root' 'install'
```